docs/source/examples/combine_techniques.md

```diff
@@ -0,0 +1,107 @@
+  language: python
+  name: python3
+---
+
+# Composing techniques: Readout Error Mitigation and Zero Noise Extrapolation
+
+Noise in quantum computers can arise from a variety of sources, and sometimes applying multiple error mitigation techniques can be more beneficial than applying a single technique alone. Here we apply a combination of readout error mitigation (REM) and zero noise extrapolation (ZNE) to a randomized benchmarking (RB) task. In REM, the inverse transition / confusion matrices are generated and applied to the noisy measurement results. In ZNE, the expectation value of the observable of interest is computed at different noise levels, and subsequently the ideal expectation value is inferred by extrapolating the measured results to the zero-noise limit. More information on the [REM](../guide/rem.md) and [ZNE](../guide/zne.md) techniques can be found in the corresponding sections of the user guide.
+
++++
+
+## Setup
+
+We begin by importing the relevant modules and libraries required for the rest of this tutorial.
+
+```{code-cell} ipython3
+import cirq
+import numpy as np
+from mitiq.benchmarks import generate_rb_circuits
+from mitiq import MeasurementResult, Observable, PauliString, raw
+```
+
+## Task
+
+We will demonstrate using REM + ZNE on RB circuits, which are generated using Mitiq's built-in benchmarking function, `generate_rb_circuits`. More information on the RB protocol is available [here](https://learn.qiskit.org/course/quantum-hardware/randomized-benchmarking). In this example we use a two-qubit RB circuit with a Clifford depth (number of Clifford groups) of 5.
+
+```{code-cell} ipython3
+circuit = generate_rb_circuits(2, 5)[0]
+```
+
+## Noise model and executor
+
+The noise in this example is a combination of depolarizing and readout errors, the latter of which are modeled as bit flips immediately prior to measurement. We use an [`Executor`](../guide/executors.md) function to execute the quantum circuit with the noise model applied.
+
+```{code-cell} ipython3
+def execute(circuit: cirq.Circuit, noise_level: float = 0.005, p0: float = 0.05) -> MeasurementResult:
+    """Execute a circuit and return a ``MeasurementResult``, with depolarizing noise of strength
+    ``noise_level`` and readout errors with probability ``p0``."""
+    measurements = circuit[-1]
+    circuit =  circuit[:-1]
+    circuit = circuit.with_noise(cirq.depolarize(noise_level))
+    circuit.append(cirq.bit_flip(p0).on_each(circuit.all_qubits()))
+    circuit.append(measurements)
+
+    simulator = cirq.DensityMatrixSimulator()
+
+    result = simulator.run(circuit, repetitions=10000)
+    bitstrings = np.column_stack(list(result.measurements.values()))
+    return MeasurementResult(bitstrings)
+```
+
+## Observable
+
+We then obtain a unmitigated expectation value of the `ZI` + `ZZ` observable. In the ideal case the result is 2, but as we will see, the unmitigated result is impacted by depolarizing and readout errors.
```

Comment:
I think this part needs a little more motivation. Just something short like
> we now have the ability to run circuits, and we wish to measure an expecation value. In this example we take the following observable...

Also you can use math mode to write out the operator $Z\otimes I + I \otimes Z$ or just $ZI + IZ$.

Reply:
Nice catch, probably a copy/paste error.